documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -0,0 +1,40 @@
+
+* Partition movement: This involves physically transferring the partition replica and its data to a new location and can take one of two forms.
+    ** Inter-broker movement: The partition replica is moved to a log directory on another broker.
+    ** Intra-broker movement: The partition replica is moved to a different log directory on the same broker.
+
+* Leadership movement: This involves switching the leader of a set of partition replicas.
+
+Cruise Control will batch up these reassignment commands and issue them to the Kafka cluster. The number of each type of these movements in these batches can effect the performance impact of the rebalance on the Kafka cluster whilst it is enacting them. Cruise control offers several configuration options for tuning the rebalance which can be set at the xref:ref-cruise-control-configuration-{context}[server] or xref:proc-generating-optimization-proposals-{context}[individual rebalance request] level. 
+
+The Cruise Control server setting can be set in the Kafka custom resource under
+`Kafka.spec.kafka.spec.cruiseControl.spec`. The individual rebalance performance configurations can be set under `KafkaRebalance.spec`. The relevant configurations are summarised below:
+
+|==========================================================================================================================
+| Server Configuration                              | KafkaRebalance Configuration            | Description | Default Value 
+
+| `num.concurrent.partition.movements.per.broker`   | `concurrentPartitionMovementsPerBroker` | 
+  The maximum number of inter-broker partition movements in each partition reassignment batch | 10
+
+| `num.concurrent.intra.broker.partition.movements` | `concurrentIntraBrokerPartitionMovements`     | 
+  The maximum number of intra-broker partition movements in each partition reassignment batch | 2
+
+
+| `num.concurrent.leader.movements`                 | `concurrentLeaderMovements`             | 
+  The maximum number of partition leadership changes in each partition reassignment batch     | 1000        
+
+
+| `default.replication.throttle`                    | `replicationThrottle`                   |
+  The bandwidth (in bytes per second) to be assigned to the reassigning of partitions         | No Limit    
+|==========================================================================================================================
+
+Changing these settings will effect the length of time the rebalance takes to complete as well as the load placed on the Kafka cluster whilst it is being enacted. Lower values will reduce the load but extend the time taken to complete and visa versa.
```

Comment:
_Suggested change:_

Changing these settings will effect the length of time the rebalance takes to complete as well as the load placed on the Kafka cluster whilst it is being enacted. Lower values will reduce the load but extend the time taken to complete and visa versa.

`>>>`

Changing the default settings affects the length of time that the rebalance takes to complete, as well as the load placed on the Kafka cluster during the rebalance. Using lower values reduces the load but increases the amount of time taken, and vice versa.